kivy/input/providers/__init__.py

```diff
@@ -52,6 +52,13 @@
     except:
         err = 'Input: LinuxWacom is not supported by your version of linux'
         Logger.exception(err)
+    try:
+        import kivy.input.providers.xinput2
+    except ImportError:
+        Logger.debug('Input: Did not import xinput2 '
```

Comment:
Missing `.` after "... after xinput2". Should be "Input: Did not import xinput2. "